src/pages/tutorials/library.tsx

```diff
@@ -70,15 +87,16 @@ export default function TutorialsLibraryPage({
 				AlternateSidebar={TutorialLibrarySidebar}
 				sidecarSlot={null}
 			>
-				<TutorialsLibraryView />
+				<TutorialsLibraryView defaultTutorials={defaultTutorials} />
 			</SidebarSidecarLayout>
 		</InstantSearch>
 	)
 }
 
-export function getStaticProps() {
+export async function getStaticProps() {
 	return {
-		props: {
+		props: stripUndefinedProperties({
+			defaultTutorials: await getTutorials(DEFAULT_SLUGS, true),
```

Comment:
Very easy with the existing tutorials methods available! 🙇

Reply:
@kendallstrautman I don't think I need the second `true` argument here for content just to render cards...accurate?

Reply:
Right, you shouldn't need the tutorial content. Perhaps its that way in learn to match the search index typing or something 🤔